### ISSUE TITLE
Astro 2.0.x has not been supported yet?

### ISSUE THREAD

**Issue by NewGyu** [vite-pwa/astro#8]
Hi, developers. Thank you for this awesome astro integration.

I tried to configure [my project](https://github.com/NewGyu/takpedia/tree/feature/pwa) that depends on astro@2.0.6 to integrate this pwa plugin , but the following error was occurred.

```shell
02:47:00 PM [astro] Unable to load /workspaces/takpedia/astro.config.mjs

 error   Cannot read properties of undefined (reading 'default')
  File:
    /workspaces/takpedia/node_modules/vite/dist/node/chunks/dep-3007b26d.js:52400:9
  Code:
    52399 |         const initModule = new AsyncFunction(`global`, ssrModuleExportsKey, ssrImportMetaKey, ssrImportKey, ssrDynamicImportKey, ssrExportAllKey, '"use strict";' + result.code + `\n//# sourceURL=${mod.url}`);
    > 52400 |         await initModule(context.global, ssrModule, ssrImportMeta, ssrImport, ssrDynamicImport, ssrExportAll);
            |         ^
      52401 |     }
      52402 |     catch (e) {
      52403 |         mod.ssrError = e;
  Stacktrace:
TypeError: Cannot read properties of undefined (reading 'default')
    at eval (/workspaces/takpedia/astro.config.mjs:21:172)
    at async instantiateModule (file:///workspaces/takpedia/node_modules/vite/dist/node/chunks/dep-3007b26d.js:52400:9)
```

After that, I also tried to reproduce it with creating new clean project with `npm create astro` command.  Same error was occurred at the project too.

The clean project's `astro.config.mjs` is simply the below.  (They were generated by `create astro` and were added only configurations for  pwa integration.)

```json:package.json
{
	"name": "nebulous-neutron",
	"type": "module",
	"version": "0.0.1",
	"scripts": {
		"dev": "astro dev",
		"start": "astro dev",
		"build": "astro build",
		"preview": "astro preview",
		"astro": "astro"
	},
	"dependencies": {
		"astro": "^2.0.14"
	},
	"devDependencies": {
		"@vite-pwa/astro": "^0.0.1"
	}
}
```

```js:astro.config.mjs
import AstroPWA from "@vite-pwa/astro"

// https://astro.build/config
export default defineConfig({
    integrations: [
        AstroPWA()
    ]
});
```

**Comment by NewGyu** [comment]
Sorry for my hasty...  #5 will solve this. 

Thank you.

**Comment by userquin** [comment]
@NewGyu released `v0.0.2`
